### ISSUE TITLE
Precipitation forcing uses non-conservative remapping

### ISSUE THREAD

**Issue by rvanharen** [eWaterCycle/ewatercycle-forecast#37]
Precipitation fluxes of the forcing are remapped using bilinear interpolation. It probably doesn't make too much of a difference, but I think we should use a conservative remapping technique here. CDO (which we already use for the remapping) supports both 1st order and 2nd order conservative remapping.

Pre-processing script is at https://github.com/eWaterCycle/ewatercycle/blob/master/cylc-suites/forecast/bin/preprocess_ensemble_forcing.sh

More details about conservative remapping at:
https://journals.ametsoc.org/doi/full/10.1175/1520-0493(1999)127%3C2204:FASOCR%3E2.0.CO;2

**Comment by RolfHut** [comment]
Hi Ronald,

personally I feel that for forecasts conservation is a more important than accuracy. The input forcing files are known to have biasses. 

For model runs and coupling, one would want conservative remapping for sure, but in a forecast I would argue that a low error in the resulting remapping (as in mean RMSE compared to truth) should be our main focus. That is why I choose bilinear in the first place.

However, my gutfeeling also tells me that the differences would be minimal and by using conservative remapping we are at least not "creating of destroying water", something reviewers are allergic to.

Rolf

**Comment by rvanharen** [comment]
I agree that the difference is probably very small, but indeed the point that I worried about was, however small, "creating or destroying water". 
I will play with it a bit and remap to different resolutions using both methods to have a better feeling of what kind of differences we talk about.
